pkgs/applications/gis/udig/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchurl, unzip, makeWrapper, jre8, libXtst, gdal }:
+let
+  inherit (stdenv.hostPlatform) system;
```

Comment:
Platforms should be sufficient for this.

Reply:
> Platforms should be sufficient for this.

What do you mean?

Reply:
Setting meta.platforms should be enough.

Reply:
Well, I've reworked this part a bit, based on https://github.com/NixOS/nixpkgs/pull/93615#issuecomment-754752145